frontend/src/pages/GamePage.js

```diff
@@ -13,13 +14,17 @@ export default function GamePage() {
   const [isRunning, setIsRunning] = useState(false);
   const [minutes, setMinutes] = useState(0);
   const [seconds, setSeconds] = useState(30);
+  const { points, getPoints, reset } = usePoints();
+
 
   useEffect(() => {
     if (letters) {
       setWord(letters);
+      getPoints()
     }
-  }, [letters]);
+  }, [letters, points]);
 
+  /** Click on button "Spielen" sets variable "isRunning" to true and activates timer with 30 seconds countdown **/
```

Comment:
I guess coaches would tell us to not use any comments in code XD

Reply:
Yes but my comments will always get updated ;)